src/Features/Core/Portable/Completion/CommonCompletionUtilities.cs

```diff
@@ -100,13 +100,12 @@ public static bool IsStartingNewWord(SourceText text, int characterPosition, Fun
         }
 
         public static async Task<CompletionDescription> CreateDescriptionAsync(
-            Workspace workspace, SemanticModel semanticModel, int position, IReadOnlyList<ISymbol> symbols, SupportedPlatformData supportedPlatforms, CancellationToken cancellationToken)
+            Workspace workspace, SemanticModel semanticModel, int position, ISymbol symbol, int overloadCount, SupportedPlatformData supportedPlatforms, CancellationToken cancellationToken)
```

Comment:
did this need to chagne?

Reply:
Did't change the original method, just moved its logic to a new helper which it now calls.